pipeline/index.js

```diff
@@ -1,34 +1,32 @@
 require('dotenv').config()
 const inquirer = require('inquirer')
-const rp = require('request-promise')
+const got = require('got')
 const chalk = require('chalk')
 
 async function getBuilds (app) {
   const auth = `Basic ${Buffer.from(process.env.BUILD_USERNAME + ':' + process.env.BUILD_TOKEN).toString('base64')}`
 
-  const { builds } = await rp({
+  const { builds } = (await got({
     url: `https://build.sys.kth.se/job/${app}/api/json?tree=builds[number,url]`,
-    method: 'GET',
     json: true,
     headers: {
       'Authorization': auth
     }
-  })
+  })).body
 
-  const buildsStatus = await Promise.all(builds.map(({ url }) => rp({
+  const buildsStatus = await Promise.all(builds.map(({ url }) => got({
     url: `${url}/api/json?`,
-    method: 'GET',
     json: true,
     headers: {
       'Authorization': auth
     }
   })))
 
   return buildsStatus
```

Comment:
If anyone (@exacs ?) can pimp the previous `Promise.all` so we don't have to fetch stuff by `body.xxx` that would be cool! I couldn't come up with anything simple...

Reply:
What do you think on this new solution…?